src/popup/router/components/Plate.vue

```diff
@@ -11,16 +9,8 @@
 
 .plate {
   width: $container-width;
-  overflow-x: hidden;
-  position: fixed;
-  z-index: 1;
-
-  .content {
-    width: #{2px + $container-width};
-    margin: -1px 0 0 -1px;
-    border: 1px solid rgba($color-border, 0.5);
-    border-radius: 0 0 10px 10px;
-    background: linear-gradient(180deg, $color-bg-3 0%, $color-black 100%);
-  }
+  box-shadow: rgba($color-border, 0.5) 0 0 0 1px;
```

Comment:
```suggestion
  box-shadow: rgba($color-border, 0.5) 0 1px;
```
this should show the border only at the bottom of node (as it is in the designs) instead of showing it around
![Screenshot 2021-03-29 at 23 27 02](https://user-images.githubusercontent.com/9007851/112895834-45a12a80-90e6-11eb-936c-899f2382a8d0.png)

Reply:
> It is consistent with the repo commit message history, in my opinion.

It may be, but it doesn't make it a good practice 😄 

Technically we can make Plate fixed on mobile (see #982), but I don't think that this code complication is reasonable, let's just fix the color of the gap between the page header and plate component on overscrolling